### ISSUE TITLE
Preseed ignores splash set to false

### ISSUE THREAD

**Issue by adamstrawson** [netson/ubuntu-unattended#7]
Hi @netson 

Firstly, thanks for this tool, really useful!

I've found that when I boot from the ISO it's sitting on the Ubuntu splash screen waiting for me to select an installation option (Run AutoInstall, Boot from X, etc)

I have splash set to False (like in your example preseed) too as per the below;

```
d-i debian-installer/splash                                 boolean     false
```

Ideally when the VM comes online I'd like it to go straight into the Auto Installation from the ISO. 

Any thoughts?

This is running Ubuntu 14.04.

Cheers!

**Comment by dgleba** [comment]
I have used this project to create my little server in a script.

I changed the grub timeout to 5 seconds and it auto installs after 5 seconds when creating the vm.
`
sed -i "s@timeout 0@timeout 50@g" $ntmp/iso_new/isolinux/isolinux.cfg
`
This is at line 173 in  https://github.com/dgleba/vamp206a/blob/master/05create-unattended-iso.sh

I felt that line 93 that I added in netson.seed didn't do the trick.
`
d-i grub-installer/timeout                                  string      6
`
https://github.com/dgleba/vamp206a/blob/master/netson.seed

I hope I am answering the need you mention above.
Let me know.

thanks

David
